photoeditor/src/main/java/ja/burhanrashid52/photoeditor/GraphicManager.kt

```diff
@@ -75,7 +75,7 @@ internal class GraphicManager(
                 mViewState.redoViewsCount - 1
             )
             if (redoView is DrawingView) {
-                return redoView.redo()
+                return redoView.redo() && (mViewState.redoViewsCount != 0)
```

Comment:
Redo still doesn't work as expected. Steps to reproduce:

* Add a brush drawing
* Add a text
* Call `undo()` twice
* Calling `redo()` returns `false`. Should return `true`.

Reply:
@wasky I changed the condition and also changed the redo logic a bit. Consider a scenario when the user does a redo action and draws another thing on top of it. I think we should clear redo history in this case.

Reply:
Yeah, I think it's LGTM as well, also the CI UI test is passing so I am assuming everything is working fine. Thanks for review and testing it out @wasky